src/random_test.cpp

```diff
@@ -150,6 +150,7 @@ int sum_of_all_options(Thd1 *thd) {
     opt_int_set(ALTER_TABLE_ENCRYPTION_INPLACE, 0);
     opt_int_set(ALTER_TABLESPACE_ENCRYPTION, 0);
     opt_int_set(ALTER_MASTER_KEY, 0);
+    opt_int_set(ALTER_ENCRYPTION_KEY, 0);
     opt_int_set(ROTATE_REDO_LOG_KEY, 0);
     opt_int_set(ALTER_DATABASE_ENCRYPTION, 0);
   }
```

Comment:
If encryption ="oracle" set ALTER_ENCRYPTION_KEY to 0.

Reply:
encryption-type is temporarily defaulted to "oracle" and it might work until then. Consider the scenario when encryption-type is defaulted to "all" and then a user runs with the option --no-encryption

Reply:
This option is disabled on upstream, and a check has been added to disable it if encryption-type="oracle"